underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -139,13 +141,14 @@ public HdfsUnderFileSystem(AlluxioURI ufsUri, UnderFileSystemConfiguration conf,
         hdfsAclProvider = (HdfsAclProvider) o;
       } else {
         LOG.warn(
-            "SupportedHdfsAclProvider is not instance of HdfsAclProvider. HDFS ACLs will not be "
-                + "supported.");
+                "{} is not instance of HdfsAclProvider. HDFS ACLs will not be "
+                        + "supported.", SupportedHdfsAclProvider.class.getCanonicalName());
       }
     } catch (Exception e) {
       // ignore
-      LOG.warn("Cannot create SupportedHdfsAclProvider. HDFS ACLs is not supported, "
-          + "Please upgrade to an HDFS version > 2.4 to enable support for ACL");
+      LOG.warn("Cannot create {}. HDFS ACLs is not supported, "
+                      + "Please upgrade to an HDFS version > 2.4 to enable support for ACL",
+              SupportedHdfsAclProvider.class.getCanonicalName());
```

Comment:
if it is not supported, the class canonical name might not exist right?

Reply:
Thanks Yuzhu! I think SupportedHdfsAclProvider is a class exist in Alluxio code base.

Reply:
Hi @shangxinli and @yuzhu , yes, it's the class exist in alluxio code, but these two class got excluded in the default profile.  So could you try to build alluxio with -PhdfsAclandActiveSync which would add these two classed back.  So actually no code change needed.